test/parallel/forall/blank-intents/array-intent-from-actual.chpl

```diff
@@ -5,8 +5,8 @@ proc explicitRef1(ref A) {
   forall i in A.domain do A[i] = i;
   [i in A.domain] A[i] = i;
   coforall i in A.domain do A[i] = i;
-  begin A = A.domain;
-  cobegin {
+  sync begin A = A.domain;
+  sync cobegin {
```

Comment:
Doesn't a `cobegin` always wait for its tasks?  (And the tasks in these `cobegin`s don't `begin` any more tasks that could outlive the `cobegin`.)

Reply:
Yes, the `sync` on the `cobegin` is technically unnecessary.